assets/src/blocks/GuestBook/GuestBookScript.js

```diff
@@ -0,0 +1,10 @@
+import {createRoot} from 'react-dom/client';
+import {GuestBookFrontend} from './GuestBookFrontend';
+
+// Fallback for non migrated content. Remove after migration.
+document.querySelectorAll('[data-render="planet4-blocks/guestbook"]').forEach(
```

Comment:
Don't we need to `hydrate` it afterwards?

Reply:
This block doesn't need hydration to work properly